go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cnoe-io/idpbuilder
 
-go 1.21.3
+go 1.22
```

Comment:
I think this needs to be `1.22.0` because that's the minimum required by K8s v0.30.0+.

Reply:
Done..!